### PR TITLE
Cleanup cloud portal article

### PR DESCRIPTION
**Pull request by eshanrnh**
Cleanup article and fix vale/markdown issues.

### CODE REVIEW

**Comment by eshanrnh** [comment]
Both the sentences are well within the 25 word count limit. Not sure why vale is still throwing an error 🤷‍♀️

**Review by sofietoft — COMMENTED** [review]
Looks great @eshanrnh !

Just added a single suggestion to the "Manage Subscriptions" part.

File: Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/index.md [hunk 1]
```diff
@@ -58,29 +74,37 @@ Managing your projects has been made even simpler with Umbraco Cloud. If you go
...
 
 In the [Settings](../../Set-Up/Project-settings) section, you will find a lot more options to configure your project.
 
 ### Manage Subscriptions
 
-Umbraco Cloud subscriptions are managed from the Umbraco Shop, which can be accessed through [umbraco.com](https://umbraco.com) or by following the **Manage Subscriptions** link from the **Profile** dropdown on the Umbraco Cloud Portal. Find out more in the [Manage Subcriptions](../../Set-Up/Manage-Subscriptions) article.
+For customers who have joined Umbraco Cloud before the 5th of May, 2021, can manage their subscriptions from Umbraco Shop. Umbraco Shop can be accessed through [Umbraco.com](https://umbraco.com). You can also click on the **Manage Subscriptions** link from the **Profile** dropdown on the Umbraco Cloud Portal. Find out more in the [Manage Subcriptions](../../Set-Up/Manage-Subscriptions/) article.
+
+For customers who have joined Umbraco Cloud after the 5th of May, 2021, can manage their subscriptions directly through Umbraco Cloud. Find out more in the [Payments on Umbraco Cloud](../../Set-Up/Manage-Subscriptions/new-shop.md) article.
```
Comment (sofietoft) [inline Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/index.md:91]:
```suggestion
Customers who have joined Umbraco Cloud after the 5th of May, 2021, can manage their subscriptions directly through Umbraco Cloud. Find out more in the [Payments on Umbraco Cloud](../../Set-Up/Manage-Subscriptions/new-shop.md) article.
```

**Review by sofietoft — COMMENTED** [review]
The additions looks great 💪 
I've made a couple of suggestions!

**Review by sofietoft — APPROVED** [review]
Looks great @eshanrnh !

Really strange that is complains about long sentences, even though there aren't any 🤷‍♀️

**Pull request merged**